packages/acs-ui-common/src/telemetry.ts

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+// TODO: Find a way to import version from ../package.json
+// This package is rooted at acs-ui-common/src so acs-ui-common/package.json is not accessible.
+const version = '0.0.0-alpha';
+
+export const sanitize = (version: string): string => {
```

Comment:
nit: add a comment what is being sanitized (not obvious that this will strip off everything after `alpha`)